server/cssfile.js

```diff
@@ -35,18 +35,21 @@ CssFile.prototype.selectorFromPosition = function(line, column){
 };
 
 CssFile.prototype.setContent = function(source, callback){
-	var messages = csslint.verify(source).messages;
-	var errors = [];
-	messages.forEach(function(msg){
-		if(msg.type == 'error'){
-			errors.push(msg);
+	const messages = stylelint.lint({code: source}).then((result) => {
+		const results = result.results
+		const errors = [];
+		results.forEach((msg) =>{
+			if(msg.type == 'error'){
+				errors.push(msg);
+				}
+			});
+		if(errors.length > 0 && callback){
+			callback(errors);
+			return;
 		}
-	});
 
-	if(errors.length > 0 && callback){
-		callback(errors);
-		return;
-	}
+	})
```

Comment:
I think it'd be a good idea to throw a `.catch(...)` on the end in case something goes wrong, don't wanna be left hanging.

Reply:
Yes, but I haven't thought how to do it. 

I thought about using the "errored" property of the stylint

"
_errored

Boolean. If true, at least one rule with an "error"-level severity registered a violation._
"
[Node.js API of Stylelint](https://stylelint.io/user-guide/usage/node-api)

I'm not that much experienced with Stylelint. I just did a little research to implement it and solve the bug I was experiencing.